Automattic/WP-Job-Manager
# Removes Mixtape and custom REST API

Fixes #1727 

#### Changes proposed in this Pull Request:

* Removes Mixtape and custom REST API implementation. This implementation was never released (hidden behind constant), deprecated in #1628 (1.32.0), and set to be removed in 1.33.0.

#### Testing instructions:

* Ensure build process works and plugin doesn't break.
* Ensure no additional references to Mixtape.
